tests/plugins/inventory/netbox/2.8.9/mocked/devices.json

```diff
@@ -0,0 +1,218 @@
+{
+    "count": 2,
```

Comment:
count = 2, but you have 3 devices in the result.
can we just use the same 4 devices as we had them for 2.3.5?

Reply:
Yeah its not relevant to this plugin but fixed so it looks less like fake data I suppose.